runtime/vm/jvminit.c

```diff
@@ -6296,6 +6296,21 @@ runExitStages(J9JavaVM* vm, J9VMThread* vmThread)
 	}
 #endif
 
+#if defined(J9VM_PROF_CONTINUATION_ALLOCATION)
```

Comment:
do we need JDK21 ifdeefs here as well

Reply:
`J9VM_PROF_CONTINUATION_ALLOCATION` flag defaults to false, and the enable option is only added for Java20/21, so I assume we don't need ifdef for Java versions